src/dodal/devices/eiger.py

```diff
@@ -35,7 +36,9 @@ def set(self, value, *, timeout=None, settle_time=None, **kwargs):
 
     STALE_PARAMS_TIMEOUT = 60
     GENERAL_STATUS_TIMEOUT = 10
-    ALL_FRAMES_TIMEOUT = 120
+    ALL_FRAMES_TIMEOUT = (
+        600  # FIXME: VMXm hack - we need to get this passed in a a param probably?
```

Comment:
Per code comment.

Reply:
Alternatively/in addition, we could make this variable default to a certain value based on the current beamline

Reply:
It's really based on the experiment. It's how long do we expect the detector to be exposing for in total